src/CTA.WebForms2Blazor/ControlConverters/ListViewControlConverter.cs

```diff
@@ -21,7 +35,29 @@ public ListViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+
+            var itemPlaceHolderIdAttr = node.Attributes.AttributesWithName("itemplaceholderid").FirstOrDefault();
+            var itemPlaceHolder = itemPlaceHolderIdAttr?.Value ?? string.Empty;
```

Comment:
Can make this more concise with:

```csharp
var itemPlaceHolderValue = node.Attributes
    .AttributesWithName("itemplaceholderid")
    .FirstOrDefault()
    ?.Value;
```

This can produce a null value, but see comment below.